build.gradle

```diff
@@ -117,8 +117,12 @@ dependencies {
 
 test {
     include '**/*Tests.class'
+    systemProperty 'tests.security.manager', 'false'
```

Comment:
Trying to understand these build.gradle settings better, does this disable security for tests?